pyuvdata/uvdata/fhd.py

```diff
@@ -604,29 +604,17 @@ def read_fhd(
         else:
             self.telescope_location_lat_lon_alt = (latitude, longitude, altitude)
 
-            # check to see if we have the layout info for this telescope.
-            # If so, don't set anything here, we'll set the values below
-            telescope_obj = uvtel.get_telescope(self.telescope_name)
-
-            if telescope_obj.Nants_telescope is None:
-                # we don't have layout info, so go ahead and set the antenna_names,
-                # antenna_numbers and Nants_telescope from the baseline info struct.
+            # we don't have layout info, so go ahead and set the antenna_names,
+            # antenna_numbers and Nants_telescope from the baseline info struct.
+            self.antenna_names = [
+                ant.decode("utf8").strip() for ant in bl_info["TILE_NAMES"][0].tolist()
```

Comment:
should the `.strip` be removed from this line? You took it out above, so I was wondering if it should be removed here too.